src/components/EditGrower.js

```diff
@@ -126,6 +141,16 @@ const EditGrower = (props) => {
             }
             onSelectChange={handleChange}
           />
+          <TextField
+            className={classes.textInput}
+            label="Image Custom URl"
+            error={!isImgURL(customUrl)}
+            onChange={(e) => {
+              if (isImgURL(e.target.value)) {
+                setCustomUrl(e.target.value);
```

Comment:
We should auto-select this image when the user enters a valid URL (by calling `handleChange('image_url', e.target.value)`. It's fair to assume that if they're entering a URL, that's the image they want to use.

Reply:
i has some issues about that.
i modify ImageScroller where the properties was imageUrl instead image_url, that works in half, in the first render, the main image was autoselected, but when i introduce a valid Custom URL, i can't making the new img into a selected one's, i don't know why.
Tomorrow with more time, i'll try to search a solution